admin/i18n/de/translations.json

```diff
@@ -13,6 +14,7 @@
     "Last": "Letzte",
     "Logging:": "Logdatei schreiben:",
     "Manufacturer": "Hersteller",
+    "Network map": "Netzwerkkarte",
```

Comment:
Vielleicht besser "Netzwerk-Übersicht"? Netzwerkkarte klingt mir sehr nach der Hardware, die sich in meinem PC befindet.